index.js

```diff
@@ -84,7 +68,7 @@ export async function deleteAsync(patterns, {force, dryRun, cwd = process.cwd(),
 		}
 
 		if (!dryRun) {
-			await rimrafP(file, rimrafOptions);
+			await rimraf(file, rimrafOptions);
```

Comment:
This package may not need to depend on rimraf at all, according to @fregante in https://github.com/sindresorhus/del/pull/161/files#r1769062773

```suggestion
			await fs.rmSync(file, { recursive: true, force: true });
```